src/ex3/chalk.js

```diff
@@ -0,0 +1,5 @@
+import chalk from "chalk";
+
+export const success = chalk.hex("4CDE90").bold;
+export const error = chalk.hex("DE4C6E").bold;
+export const blue = chalk.hex("4C5ADF").bold;
```

Comment:
cool :)
I think your color choice was great, in general the UX was very smooth, you clearly thought about giving a clear exprience to the user